index.html

```diff
@@ -0,0 +1,67 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+    <link rel="stylesheet" href="styles.css"> 
+  <header>
+  <h1 id="title">TWO RIVERS MALL</h1>
+  </header>
+  <body>
+  <p id="description"> Fill the survey below about your experience at The two rivers mall</p>
+  <form id="survey-form">
+  <div class="input-headings">
+  
+    <label for="name" id="name-label">Name </label>
+    <input id="name" type="text" name="name" placeholder="Chris Kirubi" class="form-inputs"required></input>
+  <br>
+  <label for="email" id="email-label">Email</label>
+  <input id="email" type="email" name="email"placeholder="chriskirubi@tworivers.com" class="form-inputs"required></input>
+  <br>
+  <label for="number" id="number-label">Age</label>
+  <input id="number" type="number" name="number" placeholder="23" min="18" max="120" class="form-inputs" required  ></input>
+  <br>
+  <label for="dropdown" id="select purchase">Select zone visited</label>  
```

Comment:
following the project requirement using semantic ul & li tags. they are all list items and are better displayed this way. 👍 Using li and ul is not only more semantic, but it gives meaning to the HTML nodes, helping screen readers, web crawlers, etc.